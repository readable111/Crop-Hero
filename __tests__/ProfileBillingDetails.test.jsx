import React from "react";
import {
    render,
    waitFor,
    screen,
    fireEvent,
} from "@testing-library/react-native";
import BillingDetailsProfile from "../app/profile-billingdetails"
import Colors from "../assets/Color";
import AsyncStorage from "@react-native-async-storage/async-storage";
import renderer from "react-test-renderer";
jest.useFakeTimers();

describe('Light Mode <BillingDetailsProfile/>', () =>{
    beforeEach(async () => {
        await AsyncStorage.setItem("dark_mode_setting", "false");
    });

    test('renders correctly', () =>{
        const tree = render(<BillingDetailsProfile/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})