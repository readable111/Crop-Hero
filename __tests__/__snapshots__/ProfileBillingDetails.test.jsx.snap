// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Light Mode <BillingDetailsProfile/> renders correctly 1`] = `null`;
